file-share-service-desktop-client/FileShareService.DesktopClient.csproj

```diff
@@ -20,12 +20,18 @@
 
   <ItemGroup>
     <PackageReference Include="MahApps.Metro" Version="2.4.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="5.0.11" />
```

Comment:
Can you please check if `Microsoft.AspNetCore.Http.Features` is needed

Reply:
I have to add latest package reference of **Microsoft.AspNetCore.Http.Features** to resolve vulnerable dependency of this package. This package is referenced in Unity.Microsoft.DependencyInjection package. Please see below screenshot for reference.
![image](https://user-images.githubusercontent.com/46884107/139683055-f781bab0-8beb-44bd-9461-b090fcc77723.png)

Reply:
Ahh ok, thanks for the explanation.